app/models/ext_management_system.rb

```diff
@@ -36,6 +36,7 @@ def self.supported_types_and_descriptions_hash
   has_many :ems_events,     -> { order "timestamp" }, :class_name => "EmsEvent",    :foreign_key => "ems_id"
   has_many :policy_events,  -> { order "timestamp" }, :class_name => "PolicyEvent", :foreign_key => "ems_id"
 
+  has_many :event_blacklists, :foreign_key => "ems_id", :dependent => :destroy
```

Comment:
Something about this reads funny to me.  I think the pluralization of blacklist implies it has multiple lists.  But I think what you want is something like `has_many :event_blacklist_entries` or something.